### PR TITLE
Adding note as suggested.

### PR DESCRIPTION
**Pull request by j0rt3g4**
I am solving Issue #2261 with an important note.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ The ideal for BitLocker management is to eliminate the need for IT admins to set
 
 Though much Windows BitLocker [documentation](bitlocker-overview.md) has been published, customers frequently ask for recommendations and pointers to specific, task-oriented documentation that is both easy to digest and focused on how to deploy and manage BitLocker. This article links to relevant documentation, products, and services to help answer this and other related frequently-asked questions, and also provides BitLocker recommendations for different types of computers.
 
+
+>[!IMPORTANT]
+> Microsoft BitLocker Administration and Monitoring (MBAM) capabilities would be offered from [SCCM in on-prem scenarios](https://docs.microsoft.com/microsoft-desktop-optimization-pack/mbam-v25/viewing-mbam-25-reports-for-the-configuration-manager-integration-topology) in a future.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
> Microsoft BitLocker Administration and Monitoring (MBAM) capabilities will be offered from [SCCM in on-prem scenarios](https://docs.microsoft.com/microsoft-desktop-optimization-pack/mbam-v25/viewing-mbam-25-reports-for-the-configuration-manager-integration-topology) in the future.
```

File: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md [hunk 2]
```diff
@@ -132,8 +136,10 @@ PS C:\> Enable-BitLocker -MountPoint "C:" -EncryptionMethod XtsAes256 -UsedSpace
 
 <br />
 
+
+
 <a id="powershell"></a>
-**Powershell**
+# **Powershell**
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
# **PowerShell**
```

**Comment by mypil** [comment]
@j0rt3g4  - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@DulceMontemayor - Can you please assist to check, approve and merge the content of this PR based on issue #2261?

Thank you.

**Comment by mypil** [comment]
@DulceMontemayor @AndreaBarr - We would you like to follow up. Can you please assist to check, approve and merge the content of this PR based on issue #2261?

Thank you.

**Pull request merged**